WordPress/Classes/ViewRelated/NUX/LoginEpilogueUserInfo.swift

```diff
@@ -35,6 +36,27 @@ public struct LoginEpilogueUserInfo {
     }
 
     init() {
+        // NO-OP
     }
+}
+
+
+// MARK: - LoginEpilogueUserInfo
+//
+extension LoginEpilogueUserInfo {
 
+    /// Updates the Epilogue properties, given an UserProfile instance.
+    ///
+    mutating func update(with profile: UserProfile) {
```

Comment:
Would it make sense to make this `update` method an initializer?  Its only usage is in `LoginEpilogueTableViewController`: 
```
var epilogueInfo = LoginEpilogueUserInfo()
epilogueInfo.update(with: userProfile)
```
Making it an initializer would let us get rid of the noop and reduce the work to a single line.   Just a thought.

Reply:
Ahhhha!! nice idea, thanks!!!